ui/src/error.ts

```diff
@@ -21,20 +21,30 @@ export const show = (message: string, context: unknown): void => {
   });
 };
 
-const fatalErrorWritable: svelteStore.Writable<boolean> = svelteStore.writable(
-  false
+// Fatal application errors that trigger a blue screen
+export type FatalError =
+  | { kind: FatalErrorKind.SESSION }
+  | { kind: FatalErrorKind.PROXY_EXIT; data: ipc.ProxyError };
+
+export enum FatalErrorKind {
+  SESSION = "SESSION",
+  PROXY_EXIT = "PROXY_EXIT",
```

Comment:
```suggestion
  Session = "SESSION",
  ProxyExit = "PROXY_EXIT",
```

We don't uppercase variants.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1216/commits/50a29d915751601d44386d6a3972728de467e554